routes/scripts/run/:script.js

```diff
@@ -2,13 +2,14 @@ const { send } = require('micro');
 const hasYarn = require('has-yarn');
 const execa = require('execa');
 
-module.exports = async ({ params: { script }, on }, res) => {
+module.exports = async (req, res) => {
+  const { params: { script } } = req;
   try {
     const command = hasYarn() ? 'yarn' : 'npm';
     const { stdout, kill } = execa(command, ['run', script]);
     stdout.pipe(process.stdout);
     send(res, 200, stdout);
-    on('close', kill);
+    req.on('close', kill);
```

Comment:
Probably also more readable like this in dependencies/add and remove

Reply:
It's actually a bugfix, without the `req.` `this` is undefined and _events throws an exception.